src/slider.jsx

```diff
@@ -76,11 +71,30 @@ var Slider = React.createClass({
       return !!child
     })
 
+    if (this.state.breakpoint) {
+      newProps = this.props.responsive.filter(resp => resp.breakpoint === this.state.breakpoint)[0];
+      // allow for custom unslick function instead of strictly static settings
+      unslick = newProps.settings === 'unslick' || (
+        typeof newProps.responsiveUnslick === 'function' &&
+          newProps.responsiveUnslick({children,newProps,state: this.state, props: this.props})
```

Comment:
this is the 2nd feature. allowing a function to be passed in from settings which decides when to unslick.